dhall/src/Dhall/Import.hs

```diff
@@ -970,7 +971,13 @@ loadWith expr₀ = case expr₀ of
 
 -- | Resolve all imports within an expression
 load :: Expr Src Import -> IO (Expr Src X)
-load expression = State.evalStateT (loadWith expression) (emptyStatus ".")
+load = loadWithDir "."
+
+-- | Resolve all imports within an expression, importing relative to the given
+-- directory.
+loadWithDir :: FilePath -> Expr Src Import -> IO (Expr Src X)
```

Comment:
Would `loadWithRootDirectory` be a better name?

Reply:
Maybe `loadRelativeTo`?